hudi-hadoop-common/src/test/java/org/apache/parquet/avro/TestAvroSchemaConverter.java

```diff
@@ -819,33 +832,34 @@ public void testReuseNameInNestedStructureAtSameLevel() throws Exception {
 
   @Test
   public void testUUIDType() throws Exception {
-    Schema fromAvro = Schema.createRecord(
+    HoodieSchema fromSchema = HoodieSchema.createRecord(
         "myrecord",
         null,
         null,
         false,
-        Arrays.asList(
-            new Schema.Field("uuid", LogicalTypes.uuid().addToSchema(Schema.create(STRING)), null, null)));
+        Collections.singletonList(
+            HoodieSchemaField.of("uuid", HoodieSchema.createUUID(), null, null)));
     String parquet = "message myrecord {\n" + "  required binary uuid (STRING);\n" + "}\n";
-    Schema toAvro = Schema.createRecord(
+    HoodieSchema toSchema = HoodieSchema.createRecord(
         "myrecord",
         null,
         null,
         false,
-        Arrays.asList(new Schema.Field("uuid", Schema.create(STRING), null, null)));
+        Collections.singletonList(HoodieSchemaField.of("uuid", HoodieSchema.create(HoodieSchemaType.STRING), null, null)));
 
-    testAvroToParquetConversion(fromAvro, parquet);
-    testParquetToAvroConversion(toAvro, parquet);
+    testAvroToParquetConversion(fromSchema, parquet);
+    testParquetToAvroConversion(toSchema, parquet);
 
     assertEquals(
-        COMPATIBLE, checkReaderWriterCompatibility(fromAvro, toAvro).getType());
+        COMPATIBLE, checkReaderWriterCompatibility(fromSchema.toAvroSchema(), toSchema.toAvroSchema()).getType());
```

Comment:
We can update this to `assertTrue(HoodieSchemaCompatibility.areSchemasCompatible(fromSchema, toSchema)`